app/src/main/java/com/futurice/freesound/feature/search/SearchFragment.java

```diff
@@ -170,4 +181,12 @@ private void showResults(@NonNull final List<DisplayableItem> sounds) {
         }
     }
 
+    private void showProgress(@NonNull final SearchState searchState) {
+        if (searchState.searchTriggered()) {
+            get(progressBar).setVisibility(View.VISIBLE);
```

Comment:
You could write this as:
```
get(progressBar).setVisibility(searchState.searchTriggered() ? View.VISIBLE : View.GONE);
```

Although there's probably value in writing utility class with `setVisibility(View view, boolean isVisible)` and using that.